src/components/LateralBar/LateralBar.scss

```diff
@@ -20,6 +20,11 @@
         background-color: $darkGreyColor;
         font-size: 1.1rem;
```

Comment:
I would use variables for font sizes, to avoid having a lot of different font sizes in the app, and forcing to use standards. For example:

$fontSizeNormal: 1rem;
$fontSizeSmall: .8rem;
$fontSizeLarge: 1.2rem;
$fontSizeExtraLarge: 2rem;

Adapt it to your needs :)